<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GrabCut</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="5">
    <widget class="QRadioButton" name="FG_btn">
     <property name="text">
      <string>Foreground</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="open_btn">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="run_btn">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>run</string>
     </property>
    </widget>
   </item>
   <item row="2" column="9">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>372</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="E_btn">
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>erase</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>scribble size : </string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="rect_btn">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>rectangle</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QSpinBox" name="scribble_size_spin"/>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="scribble_btn">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>scribble</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QRadioButton" name="BG_btn">
     <property name="text">
      <string>Background</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="10">
    <widget class="QLabel" name="palette_map">
     <property name="maximumSize">
      <size>
       <width>2000</width>
       <height>2000</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../공부/image.qrc">:/jpg/image.jpg</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../공부/image.qrc"/>
  <include location="../../../../.designer/backup/image.qrc"/>
 </resources>
 <connections/>
</ui>
